packages/web/src/containers/decrease-liquidity-container/DecreaseLiquidityContainer.tsx

```diff
@@ -0,0 +1,56 @@
+import DecreaseLiquidity from "@components/decrease/decrease-liquidity/DecreaseLiquidity";
+import { useDecreaseHandle } from "@hooks/decrease/use-decrease-handle";
+import { useDecreasePositionModal } from "@hooks/decrease/use-decrease-position-modal";
+import React from "react";
+
+const DecreaseLiquidityContainer: React.FC = () => {
+  const {
+    tokenA,
+    tokenB,
+    fee,
+    minPriceStr,
+    maxPriceStr,
+    rangeStatus,
+    aprFee,
+    priceRangeSummary,
+    percent,
+    setPercent,
+    pooledTokenInfos,
+  } = useDecreaseHandle();
+
+  const { openModal } = useDecreasePositionModal({
+    tokenA,
+    tokenB,
+    swapFeeTier: `FEE_${fee}` as any,
+    minPriceStr,
+    maxPriceStr,
+    rangeStatus,
+    percent,
+    pooledTokenInfos,
+  });
+
+  const onSubmit = () => {
+    openModal();
+  };
+
+  if (!tokenA || !tokenB) return null;
```

Comment:
Why do you return null instead of empty component?

Reply:
Done